pkgs/top-level/all-packages.nix

```diff
@@ -3431,6 +3431,8 @@ with pkgs;
 
   macchanger = callPackage ../os-specific/linux/macchanger { };
 
+  mackup = callPackage ../tools/backup/mackup { };
```

Comment:
You can use `python2Packages.callPackage` and specify the dependencies as arguments instead of `python2Packages`.

Reply:
Hey, thanks for the feedback. I'll try to get to this in the next day or two.